infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -706,7 +706,7 @@ private boolean clusterRoleChanged(TopologyDescriptor discoveredTopology) {
      * @param discoveredTopology new discovered topology
      */
     private void onStandbyClusterAddRemove(TopologyDescriptor discoveredTopology) {
-        log.debug("Standby Cluster has been added or removed from topology={}", discoveredTopology);
```

Comment:
Already logged topology change when received notification from cluster manager, and topology descriptor is not formatted and hard to read.

for reference: CorfuReplicationDiscoveryService.java line 655 
public void processTopologyChangeNotification(DiscoveryServiceEvent event)